redhat-cop/openshift-applier
# Introduce 'exclude_tags'

#### What does this PR do?
Adds the skip_tags option to be used in the same manner as filter_tags.

#### How should this be tested?
Specify the skip_tags variable when running openshift_applier against an inventory which has tags set, verify that no objects with the given tag were applied. You can also verify that the filter_tags option still works along with neither option. 

#### Is there a relevant Issue open for this?
resolves #118 

#### Who would you like to review this?
cc: @redhat-cop/openshift-applier
